executor/ddl.go

```diff
@@ -152,6 +155,30 @@ func (e *DDLExec) executeCreateTable(s *ast.CreateTableStmt) error {
 	return errors.Trace(err)
 }
 
+func (e *DDLExec) executeCreateView(s *ast.CreateViewStmt) error {
+	ident := ast.Ident{Schema: s.ViewName.Schema, Name: s.ViewName.Name}
+
+	// SelectFields represents the field expression or field AsName in SelectStmt
+	// and saves it as a string to establish the mapping relationship of ColList
+	var selectFields []string
+	cols := e.schema.Columns
+	for _, col := range cols {
+		selectFields = append(selectFields, col.ColName.L)
+	}
+
+	// The column_list in Create View statement could be null, so if Cols is nil,
+	// it saves select_statement field expression or field AsName
+	if s.Cols == nil {
+		s.Cols = selectFields
+	}
+
+	err := sessionctx.GetDomain(e.ctx).DDL().CreateView(e.ctx, ident, s.Cols, selectFields, s.Select.Text())
+	if infoschema.ErrTableExists.Equal(err) {
```

Comment:
Why check this error? Could we save this if?

Reply:
Because do create view job is like do create table, it needs to check this error